<launch>

  <!--<include file="$(find openni2_launch)/launch/openni2.launch">-->
  <!--  <arg name="depth_registration" value="true" />-->
  <!--</include>-->

  <include file="$(find realsense2_camera)/launch/rs_camera.launch">
    <param name="initial_reset" value="true" />
  </include>

  <node pkg="elastic_bridge" type="elastic_node" name="elastic_node" output="screen" launch-prefix="gdb -ex run --args" >
    <param name="AUTOSTART" type="bool" value="true" />
    <param name="TOPIC_IMAGE_COLOR" type="string" value="/camera/color/image_raw" />
    <param name="TOPIC_IMAGE_DEPTH" type="string" value="/camera/depth/image_rect_raw" />
    <param name="TOPIC_CAMERA_INFO" type="string" value="/camera/color/camera_info" />
    <param name="PERIODIC_WORLD_PUBLICATION" type="int" value="90" />
  </node>

</launch>

